templates/agent/nfsclient.conf.erb

```diff
@@ -1,3 +1,8 @@
-# nfs
-UserParameter=nfs.mounted,/usr/bin/stat -f /var/www | /bin/grep -cio 'Type: nfs'
-UserParameter=nfs.available,(/usr/bin/test -f /var/www/NE_BRISI_ME.txt && /bin/echo 1) || /bin/echo 0
+##############################################################
+#                                                            #
+#   This file is managed by Puppet - DO NOT EDIT MANUALLY    #
+#                                                            #
+##############################################################
```

Comment:
Can you remove empty lines and remove all these comment-lines. It's enough to keep just 1-line header (`Managed by puppet`).

Reply:
Done.